notebooks/data-sources/TestGrid/metrics/probability_to_fail.ipynb

```diff
@@ -0,0 +1,1091 @@
+{
```

Comment:
<p>related to the above comment: but what does predicting a -1 label mean here? should we rename <code>pass_or_fail</code> then to something like <code>pass_fail_or_neither</code> depending on what it signifies as its a bit misleading.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/310/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>We are just assigned <code>-1</code>  to the NaN records. Because the model needs a numeric value. </p>

Reply:
<p>Why don't you simply remove the NaN records? They represent a number of different label types and might make learning a probability of pass or fail more difficult. </p>

Reply:
<p>Sure. We will remove that</p>